src/rest.ts

```diff
@@ -149,6 +151,15 @@ ${queryParams
         console.log('Response', loggedResponse)
         console.groupEnd()
       },
+
+      getMetaInfo() {
+        return {
+          header: `[rest] ${method} ${mask.toString()}`,
+          mask,
+          method,
```

Comment:
Should `method` be returned here?

Reply:
I suppose we can drop it. Originally I thought it could be used in the REPL, but then realized it's expected/actual paths comparison at the moment, so a request's method is not of much use.

Reply:
I was only asking it because the property isn't required by the `RequestHandlerMetaInfo ` type.

Reply:
Oh, a great point. I wasn't sure how to approach the meta data type-wise as I can see a potential room for arbitrary properties in case of custom request handlers. I need to type that better 💯